osquery/tables/system/windows/windows_search.cpp

```diff
@@ -0,0 +1,399 @@
+// osquery/tables/system/windows/windows_search.cpp
+
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <windows.h>
+#include <atlbase.h>
+#include <searchapi.h>
+#include <atldbcli.h>
+#include <codecvt>
+#include <comutil.h>
+#include <osquery/core/core.h>
+#include <osquery/core/tables.h>
+#include <osquery/logger/logger.h>
+#include <osquery/utils/conversions/windows/strings.h>
+#include <osquery/utils/conversions/windows/windows_time.h>
+#include <osquery/utils/conversions/join.h>
+#include <osquery/utils/scope_guard.h>
+#include <sstream>
+#include <strsafe.h>
+#include <vector>
+
+#pragma comment(lib, "comsuppw.lib")
```

Comment:
Required link libraries should be moved to Cmakelist to better tracking of dependencies. See [this](https://github.com/osquery/osquery/pull/2904/files) PR for examples of achieving this.